### PR TITLE
add createAccountNegSpec test

### PR DESCRIPTION
**Pull request by vdemnyk**
Add create account negative test

### CODE REVIEW

File: conf.js [hunk 1]
```diff
@@ -7,23 +7,22 @@ exports.config = {
     jasmineNodeOpts: {
         defaultTimeoutInterval: 90000
       },
-    specs: ['./spec/loginSpec.js','./spec/createAccountSpec.js'],
+    specs: ['./spec/loginSpec.js','./spec/createAccountSpec.js','./spec/createAccountNegSpec.js'],
     onPrepare: function(){
         console.log('Start of tests execution');
-        browser.waitForAngularEnabled(false);
         browser.manage().timeouts().implicitlyWait(4000);
```
Comment (nbabii) [inline conf.js]: Move this into `beforeEach()` also

File: pages/login.page.js [hunk 2]
```diff
@@ -58,6 +59,22 @@ class LoginPage extends WebPage {
         })();  
     }
 
+    async alertSignupIsVisible() {
+        let ec = await browser.ExpectedConditions;
+        await browser.wait(ec.visibilityOf($('#create_account_error>ol>li')), 4000);
```
Comment (nbabii) [inline pages/login.page.js]: Create `waitForVisible()` method in the base element, using the same approach.
After, here, you can simply use `this.getAlertSignupBlock().waitForVisible()`

File: pages/register.page.js [hunk 3]
```diff
@@ -34,6 +35,8 @@ class RegisterPage extends WebPage {
         await allure.createStep(`Type ${address}, ${city}, ${postcode}; select state`, async () => {
             await this.getAddressElement().sendKeys(address);
             await this.getCityElement().sendKeys(city);
+            let ec = await browser.ExpectedConditions;
+            await browser.wait(ec.visibilityOf($('#uniform-id_state')), 4000);
```
Comment (nbabii) [inline pages/register.page.js]: Same as above use `waitForVisible()`

**Review by nbabii — CHANGES_REQUESTED** [review]

File: conf.js (hunk 1 above)
Comment (vdemnyk) [reply conf.js]: Moved

File: pages/login.page.js (hunk 2 above)
Comment (vdemnyk) [reply pages/login.page.js]: Done

File: pages/register.page.js (hunk 3 above)
Comment (vdemnyk) [reply pages/register.page.js]: Done

File: conf.js (hunk 1 above)
Comment (nbabii) [reply conf.js]: `browser.manage().timeouts().implicitlyWait(4000);` Move this into beforeEach()

**Review by nbabii — CHANGES_REQUESTED** [review]

**Review by nbabii — APPROVED** [review]

**Pull request merged**